@shared/api/internal.ts

```diff
@@ -1356,13 +1348,6 @@ export const loadSettings = (): Promise<
     type: SERVICE_TYPES.LOAD_SETTINGS,
   });
 
-export const getBlockedDomains = async () => {
-  const resp = await sendMessageToBackground({
```

Comment:
blockedDomains were used to check the domain of an asset against a list of malicious domains. Blockaid will handle this check now